renku/cli/.gitlab-ci.yml

```diff
@@ -1,4 +1,10 @@
-# renku: v1.dev20180619
+# renku: v1.dev20180723
+
+variables:
+  GIT_STRATEGY: clone
```

Comment:
@leafty is probably right, this should be changed to fetch

Reply:
changed